lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/KbvEvidenceCheckServiceTest.java

```diff
@@ -0,0 +1,219 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.cri.common.library.util.EventProbe;
+import uk.gov.di.ipv.cri.kbv.api.domain.CheckDetail;
+import uk.gov.di.ipv.cri.kbv.api.domain.ContraIndicator;
+import uk.gov.di.ipv.cri.kbv.api.domain.KBVItem;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuality;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvResponseMode;
+import uk.gov.di.ipv.cri.kbv.api.domain.VerifiableCredentialConstants;
+import uk.gov.di.ipv.cri.kbv.api.service.fixtures.TestFixtures;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.verify;
+import static uk.gov.di.ipv.cri.kbv.api.service.KbvEvidenceCheckService.METRIC_DIMENSION_KBV_VERIFICATION;
+
+@ExtendWith(MockitoExtension.class)
+class KbvEvidenceCheckServiceTest implements TestFixtures {
+    private KbvEvidenceCheckService kbvEvidenceCheckService;
+    @Mock private EventProbe mockEventProbe;
+    private final ObjectMapper objectMapper =
+            new ObjectMapper()
+                    .registerModule(new Jdk8Module())
+                    .registerModule(new JavaTimeModule());
+
+    @BeforeEach
+    void setUp() {
+        kbvEvidenceCheckService = new KbvEvidenceCheckService(objectMapper, mockEventProbe);
+    }
+
+    @Test
```

Comment:
Please check jacoco coverage report to ensure sufficient code, branches etc have been covered